Examples/SnapinsSDKKotlinExample/app/src/javaImplDebug/java/com/salesforce/snapinssdkexample/activities/settings/CaseSettingsActivity.java

```diff
@@ -0,0 +1,31 @@
+package com.salesforce.snapinssdkexample.activities.settings;
+
+import android.os.Bundle;
+import android.os.PersistableBundle;
+import android.preference.PreferenceFragment;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import com.salesforce.snapinssdkexample.R;
+
+public class CaseSettingsActivity extends AppCompatActivity {
+    public static final String KEY_COMMUNITY_URL = "pref_case_community_url";
```

Comment:
We could probably store all of these KEYS under the `main` variant and share them between `javaDebugImpl` and `kotlinImplDebug`?

Reply:
I'm not sure how to do this either.

Reply:
It was just a thought, these are fine to stay as is.